app/selectors/transactionController.ts

```diff
@@ -6,11 +6,16 @@ import { selectPendingSmartTransactionsBySender } from './smartTransactionsContr
 const selectTransactionControllerState = (state: RootState) =>
   state.engine.backgroundState.TransactionController;
 
-const selectTransactionsStrict = createSelector(
+export const selectTransactionsStrict = createSelector(
   selectTransactionControllerState,
   (transactionControllerState) => transactionControllerState.transactions,
 );
 
+export const selectTransactionBatchesStrict = createSelector(
```

Comment:
```suggestion
const selectTransactionBatchesStrict = createSelector(
```
Minor, removing import if we are not using it outside?